stable/pipelines/values.yaml

```diff
@@ -138,7 +142,7 @@ pipelines:
 
   api:
     image:
-      registry: docker.bintray.io
+      # registry:
```

Comment:
why this was added?

Reply:
we made this change to sync with all charts (jfrog charts and external charts)